tcms/rpc/api/bug.py

```diff
@@ -37,6 +37,8 @@ def details(url, **kwargs):
     if not result:
         request = kwargs.get(REQUEST_KEY)
         tracker = tracker_from_url(url, request)
+        if not tracker:
+            return None
```

Comment:
To answer my question below I think here you should return a dict which has all of the necessary keys but empty values so that clients won't have to do their own checking. If you insist on returning None it should be documented and then tested.

Reply:
I don't insist on returning `None`. How about an empty dict - `{}`?

Reply:
`assignPopoverData` in the JS source will try using the keys from this dict and IDK how it will break if the result is None or empty dict b/c for the user there won't the the keys it expects.

Reply:
from the browser console:
```
>> const emptyDict = {}
>> emptyDict.something;
undefined
```
```
>> const undef = undefined;
>> undef.something;
Uncaught TypeError: undef is undefined
```
accessing not existing field on an object/dict, will just return undefined, hence returning an empty should not break anything. I think that even if we return `None` it gets serialized as `{}`, so there should be no issues here.